rasa/importers/utils.py

```diff
@@ -10,3 +13,20 @@ def training_data_from_paths(paths: Iterable[Text], language: Text) -> TrainingD
     merged_training_data = TrainingData().merge(*training_datas)
     merged_training_data.fill_response_phrases()
     return merged_training_data
+
+
+async def story_graph_from_paths(
+    files: List[Text],
+    domain: Domain,
+    interpreter: NaturalLanguageInterpreter = RegexInterpreter(),
+    template_variables: Optional[Dict] = None,
+    use_e2e: bool = False,
+    exclusion_percentage: Optional[int] = None,
+) -> StoryGraph:
+
+    from rasa.core.training import loading
+
+    story_steps = await loading.load_data_from_files(
+        files, domain, interpreter, template_variables, use_e2e, exclusion_percentage
+    )
+    return StoryGraph(story_steps)
```

Comment:
Actually, I've chosen this places because of the symmetry with the `NLU` loading in the same place.
The `rasa/core/training` has a `loading.py`, that's imported here.
So, the "high-level" class for the NLU is `TrainingData`, and for core it's `StoryGraph`, both are created here.

I see the confusion though, and I plan to address that with next PR that will split loading of NLU and Core.